### PR TITLE
Feature/create discussion posts

### PR DESCRIPTION
**Pull request by codiebeulaine**
@miltontony @justinvdm ready for review

### CODE REVIEW

File: mentorship/mentor/serializers.py [hunk 1]
```diff
@@ -16,6 +16,20 @@ def get_image(self, obj):
         return get_image_url(obj.image)
 
 
+class PostSerializer(serializers.ModelSerializer):
+    child_comments = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Post
+        fields = ('comment', 'parent', 'object_pk', 'child_comments')
```
Comment (miltontony) [inline mentorship/mentor/serializers.py]: I think this should be called `num_child_comments`

Comment (codiebeulaine) [inline mentorship/mentor/serializers.py]: fixed

File: mentorship/mentor/serializers.py [hunk 2]
```diff
@@ -16,6 +16,20 @@ def get_image(self, obj):
...
+
+    class Meta:
+        model = Post
+        fields = ('comment', 'parent', 'object_pk', 'num_child_comments')
+        read_only_fields = (
+            'site', 'content_type', 'user', 'email', 'submit_date',
+            'num_child_comments')
+
+    def get_num_child_comments(self, obj):
+        return Post.objects.filter(parent=obj.pk).count()
```
Comment (miltontony) [inline mentorship/mentor/serializers.py]: I think you should be able to just do `self.posts.count()`

Comment (miltontony) [inline mentorship/mentor/serializers.py]: or `self.posts.all().count()` not sure which

Comment (miltontony) [inline mentorship/mentor/serializers.py]: or it might be `post_set`

Comment (codiebeulaine) [inline mentorship/mentor/serializers.py]: @miltontony I don't understand that?

Comment (codiebeulaine) [inline mentorship/mentor/serializers.py]: @miltontony ah I think I do, but it's not working

**Comment by miltontony** [comment]
👍

**Pull request merged**